webpack/components/AnsibleRoles/components/AnsibleRolesImportButton.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { importUrl } from '../AnsibleRolesPageHelpers';
+import { showToast } from '../../../toastHelper';
+import {
+  formatError,
+  joinErrors,
+} from '../../AnsibleHostDetail/components/AnsibleVariableOverrides/EditableActionHelper';
+
+export const AnsibleRolesImportButton = () => {
+  let dropdownItems;
+
+  const {
+    response: { results },
+    status,
+  } = useAPI('get', '/api/v2/smart_proxies', {
+    params: { feature: 'Ansible' },
```

Comment:
Sorry, this should have been a search query
```suggestion
    params: { search: 'feature=Ansible' },
```

Reply:
Yeah, this works now.